tests/framework/defs.py

```diff
@@ -26,6 +26,9 @@
 # Default test session root directory path
 DEFAULT_TEST_SESSION_ROOT_PATH = "/srv"
 
+# Default test session artifacts path
```

Comment:
this should go in a separate commit

Reply:
what exactly? hard  to tell just from 1 line

Reply:
I mean all the changes in this file look like a preparation for the integration test, but are not an immediate part of it. Having a commit with an explanation of the change would be nice.

Reply:
I don't think splitting changes in such a small parts makes a lot of sense in this case.